### PR TITLE
Revert "Create the database"

### PR DESCRIPTION
**Pull request by stevengoossensB**
Reverts MISP/misp-docker#79

Hi @adulau I think we should revert this PR back to the @xme version. I have been continuing the work on a new container instance which uses the MISP install script, but I think we should take this as a separate track/container implementation in a different Git project.

### CODE REVIEW

File: web/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,191 @@
...
+USER www-data
+RUN git clone https://github.com/MISP/MISP.git
+WORKDIR /var/www/MISP
+RUN git checkout tags/$(git describe --tags `git rev-list --tags --max-count=1`) && \
+    git config core.filemode false && \
+    git submodule update --init --recursive && \
+    git submodule foreach --recursive git config core.filemode false
+
+USER root
+RUN pip3 install git+https://github.com/STIXProject/python-stix.git \
```
Comment (helinko) [inline web/Dockerfile:54]: PIP is now too old which breaks the build, see question 2 here: https://pypi.org/project/opencv-python/

This should fix it, tested locally and seems to work:

```suggestion
RUN pip3 install --upgrade pip
RUN pip3 install git+https://github.com/STIXProject/python-stix.git \
```

**Review by helinko — APPROVED** [review]
Tested and it builds and runs with a minimal setup

**Comment by jellard1** [comment]
Merge it! Go, go, go!
-says the guy who just discovered this issue 15 minutes ago. It's been broken since July? Well looks like I ran into this right as you were fixing it!  Can you merge @helinko or only @stevengoossensB can?

**Comment by helinko** [comment]
I don't have commit rights and neither does @stevengoossensB, I assume. @adulau at least has. In the meantime you can just use the branch from Steven's fork of the repo.

**Comment by helinko** [comment]
Only now tested the nginx version and that seems broken. Do you know whether it was like that before the initial merge?

**Comment by stevengoossensB** [comment]
What error message are you getting? I'm not using any of the compose versions since I'm running it on Azure and have created my template there and use the infra there for reverse proxying.

**Comment by stevengoossensB** [comment]
> 
> 
> Only now tested the nginx version and that seems broken. Do you know whether it was like that before the initial merge?

Nvm, I found an error in the NGINX config where it redirected to misp_web, while it should be web. Could you retry?

**Comment by helinko** [comment]
Yeah I think that's an error, as curling from the proxy container returns 400 for misp_web but web works as expected. However, there's something also broken in the nginx config since it still goes to the "Proxy Backend Not Found" custom error page.

I don't know enough about nginx to see right away what's wrong, but I'll continue debugging tomorrow if no one else has found a fix.

**Comment by helinko** [comment]
The http->https rewrite rule is broken and I have a fix for it, plus a couple of other improvements. It's not really in the scope of this PR though, since you haven't touched that part at all. So maybe I'll just make a new PR about it?

**Comment by adulau** [comment]
Let us know if this is fine ;-)

File: web/Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ FROM ubuntu:focal
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt update -y && apt install wget sudo -y
 # This will install MISP Core
-RUN wget -O /tmp/INSTALL.sh https://raw.githubusercontent.com/MISP/MISP/2.4/INSTALL/INSTALL.sh
+RUN wget -O /tmp/INSTALL.sh https://raw.githubusercontent.com/stevengoossensB/MISP/2.4/INSTALL/INSTALL.sh
```
Comment (helinko) [inline web/Dockerfile]: Is this intentional?

Comment (helinko) [reply web/Dockerfile]: Never mind, it's overridden by a later commit so it doesn't actually change the current state of the file.

**Pull request merged**